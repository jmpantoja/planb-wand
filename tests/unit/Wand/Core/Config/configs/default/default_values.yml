---
actions:
  readme:
    group: metainfo
    classname: \PlanB\Wand\Core\File\File
    params:
      template: '@wand.metainfo.readme.twig'
      target: README.md
      action: create # delete, create
      chmod: 0644
tasks:
  init:
    description: wubba lubba dub dub
    actions:
      readme:

  taskB:
    classname: \PlanB\Wand\Core\Task\Task
    description: wubba lubba dub dub
    actions:
      readme:
