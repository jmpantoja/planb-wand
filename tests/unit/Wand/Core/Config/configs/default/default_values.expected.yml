---
tasks:
  init:
    classname: PlanB\Wand\Core\Task\Task
    description: wubba lubba dub dub
    actions:
      readme: []